core/src/main/java/io/undertow/server/protocol/framed/AbstractFramedChannel.java

```diff
@@ -813,14 +815,15 @@ public boolean isReceivesResumed() {
      * Forcibly closes the {@link io.undertow.server.protocol.framed.AbstractFramedChannel}.
      */
     @Override
-    public void close() throws IOException {
+    public synchronized void close() throws IOException {
```

Comment:
Oops, there is no need for this synchronized here. I'm submitting a new PR to remove this